src/sentry/static/sentry/app/views/settings/project/projectKeys/details/keyRateLimitsForm.tsx

```diff
@@ -97,7 +98,16 @@ class KeyRateLimitsForm extends React.Component<Props> {
                       number of events processed.`
                   )}
                 </PanelAlert>
-                {!hasFeature && renderDisabled({organization, project, features})}
+                {!hasFeature &&
+                  typeof renderDisabled === 'function' &&
+                  renderDisabled({
+                    organization,
+                    project,
+                    features,
+                    renderDisabled,
```

Comment:
it passes itself to the renderFn?

Reply:
Yeah I think this did become.. somewhat confusing. It's because in this case renderDisabled and `children` share render props. I could maybe think about this a bit more but the issue is above on line +85, where it passes renderDisabled in the children.

Let me try and look a bit closer.

Reply:
Okay, I got rid of this weridness.